### PR TITLE
Add lendong pool tvl for sft protocol

### PR DESCRIPTION
**Pull request by Sunnight-wj**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/sft-protocol exports TVL: 
        
 
 
    filecoin                  4.33 M
    
    total                    4.33 M

File: projects/sft-protocol/index.js [hunk 1]
```diff
@@ -40,7 +43,9 @@ module.exports = {
     filecoin: {
         tvl: async (_, _1, _2, { api }) => {
 
-            let balances = {};
+            const lendingPoolTvl = await api.call({ target: LENDING_POOL_ADDRESS, abi: TOTAL_SUPPLY_ABI, params: [] });
+            api.add(ADDRESSES.null, lendingPoolTvl);
```
Comment (g1nt0ki) [inline projects/sft-protocol/index.js:47]: why is totalSupply added to tvl and also token balance in lending pool? shouldnt we include only the token balance in lending pool? isnt the borrowed amount already counted under active miners?

**Comment by llamatester** [comment]
The adapter at projects/sft-protocol exports TVL: 
        
 
 
    filecoin                  4.32 M
    
    total                    4.32 M

File: projects/sft-protocol/index.js (hunk 1 above)
Comment (Sunnight-wj) [reply projects/sft-protocol/index.js:47]: > why is totalSupply added to tvl and also token balance in lending pool? shouldnt we include only the token balance in lending pool? isnt the borrowed amount already counted under active miners?

hey guy, please check again and merge it if no problem.

**Comment by Sunnight-wj** [comment]
as no one deal this PR, I close it and resubmit.

**Pull request closed without merging**